astropy/cosmology/_src/core.py

```diff
@@ -89,9 +90,13 @@ class CosmologyError(Exception):
 #       See https://peps.python.org/pep-0712/
 @dataclass(frozen=True, slots=True)
 class _NameField:
+    """Descriptor for the `Cosmology.name` field."""
+
```

Comment:
This might be a good change, but it's not related to typing.

Reply:
removed.